modules/ServerAPI/src/server/routes/AuthorRoutes.coffee

```diff
@@ -681,6 +681,28 @@ exports.updateAuthorAndRolesInternal = (author, user, callback) ->
 																		else
 																			callback null, response
 
+exports.syncRoles = (author, rolesToAdd, rolesToDelete, callback) =>
```

Comment:
This is the function we removed in a preceding PR that broken saml sync roles. We just added it back here.